components/style/properties/shorthand/font.mako.rs

```diff
@@ -4,17 +4,18 @@
 
 <%namespace name="helpers" file="/helpers.mako.rs" />
 
-<%helpers:shorthand name="font" sub_properties="font-style font-variant font-weight
+<%helpers:shorthand name="font" sub_properties="font-style font-variant font-weight font-stretch
                                                 font-size line-height font-family"
                     spec="https://drafts.csswg.org/css-fonts-3/#propdef-font">
-    use properties::longhands::{font_style, font_variant, font_weight, font_size,
+    use properties::longhands::{font_style, font_variant, font_weight, font_stretch, font_size,
                                 line_height, font_family};
 
     pub fn parse_value(context: &ParserContext, input: &mut Parser) -> Result<Longhands, ()> {
         let mut nb_normals = 0;
         let mut style = None;
         let mut variant = None;
         let mut weight = None;
+        let mut stretch = None;
         let size;
         loop {
             // Special-case 'normal' because it is valid in each of
```

Comment:
There is a special case for `normal` values. Font stretch have also `normal` value as initial value. You should update this comment to include font-stretch and change [this line](https://github.com/servo/servo/pull/15134/files#diff-29d4c395421b6645108dad1c5900daf4L52) to cover font-stretch too.